twitter/api.py

```diff
@@ -3008,28 +3010,52 @@ def PostDirectMessage(self,
           text: The message text to be posted.
           user_id:
             The ID of the user who should receive the direct message.
+          media_file_path:
+            The file path to the media to be posted
+          media_type:
+            The media type. Accepted media types: dm_image, dm_gif or dm_video
           return_json (bool, optional):
             If True JSON data will be returned, instead of twitter.DirectMessage
         Returns:
           A twitter.DirectMessage instance representing the message posted
         """
         url = '%s/direct_messages/events/new.json' % self.base_url
-
         # Hack to allow some sort of backwards compatibility with older versions
         # part of the fix for Issue #587
         if user_id is None and screen_name is not None:
             user_id = self.GetUser(screen_name=screen_name).id
 
+        # Default
+        message_data_value = {
+            'text': text
+        }
+        if media_file_path is not None:
+            try:
+                image = open(media_file_path, 'rb')
```

Comment:
Since you can send an image or video:

```suggestion
                media = open(media_file_path, 'rb')
```